api/hearings/data/serviceHearingValues.mock.data.ts

```diff
@@ -1,20 +1,22 @@
 /* tslint:disable:object-literal-sort-keys */
 import {
   CategoryType,
+  HearingChannelEnum,
   HMCLocationType,
   MemberType,
   PartyType,
   RequirementType,
   UnavailabilityType
 } from '../models/hearings.enum';
-import {ServiceHearingValuesModel} from '../models/serviceHearingValues.model';
+import { ServiceHearingValuesModel } from '../models/serviceHearingValues.model';
 
 export const SERVICE_HEARING_VALUES: ServiceHearingValuesModel = {
   hmctsServiceID: 'BBA3',
   hmctsInternalCaseName: 'Jane and Smith vs DWP',
   publicCaseName: 'Jane and Smith vs DWP',
   autoListFlag: false,
   hearingType: 'BBA3-DIR',
+  hearingChannel: [HearingChannelEnum.ONPPR],
```

Comment:
Probably we also need to create test sceanrios for NON-PAPER hearing channel pass from service.